srcpkgs/wxWidgets/template

```diff
@@ -1,21 +1,25 @@
 # Template file for 'wxWidgets'
 pkgname=wxWidgets
-version=3.0.5.1
+version=3.1.4
 revision=1
-configure_args="--enable-unicode --with-opengl --with-sdl --with-libmspack
- --with-libnotify --enable-mediactrl --with-gtk=2 --disable-webview"
 build_style=gnu-configure
+configure_args="--disable-option-checking --disable-webview
+ --enable-mediactrl --enable-unicode
+ --with-gtk=2 --with-libmspack --with-libnotify --with-opengl --with-sdl"
```

Comment:
Is it possible to switch to gtk3 here?

Reply:
There is another package for Gtk3.

Reply:
It seems most of the packages can be recompiled for wxGtk3, I will try to do this in a separate branch.